{
  "id": "CVE-2020-11538",
  "package_name": "pillow",
  "fix_state": "unknown",
  "cpes": [
    "cpe:2.3:a:python:pillow:*:*:*:*:*:*:*:*"
  ],
  "version_constraint": "<= 7.0.0",
  "fixed_in_versions": "null",
  "severity": "High",
  "description": "In libImaging/SgiRleDecode.c in Pillow through 7.0.0, a number of out-of-bounds reads exist in the parsing of SGI image files, a different issue than CVE-2020-5311.",
  "urls": [
    "https://github.com/python-pillow/Pillow/pull/4504",
    "https://github.com/python-pillow/Pillow/pull/4538",
    "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/BEBCPE4F2VHTIT6EZA2YZQZLPVDEBJGD/",
    "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/HOKHNWV2VS5GESY7IBD237E7C6T3I427/",
    "https://pillow.readthedocs.io/en/stable/releasenotes/7.1.0.html",
    "https://pillow.readthedocs.io/en/stable/releasenotes/index.html",
    "https://usn.ubuntu.com/4430-1/",
    "https://usn.ubuntu.com/4430-2/"
  ]
}
